public/manifest.json

```diff
@@ -26,7 +26,7 @@
       "js": ["build/contentScript.js"]
     },
     {
-      "matches": ["https://console.brahma.fi/account/*/kernel"],
+      "matches": ["https://console.brahma.fi/console/*/kernel"],
```

Comment:
shouldn't this be more generic as
 "matches": ["https://console.brahma.fi/console/*"],